Translated 'Additional.md' file into French (#503)
trigaten/Learn_Prompting: i18n/fr/docusaurus-plugin-content-docs/current/additional.md

---
sidebar_position: 3000
---

# 🛸 Ressources supplémentaires

## Ressources

> Une vidéo d'introduction fantastique à PE : https://youtube.com/watch?v=dOxUroR57xs&feature=shares

Reviewer: ```suggestion
Une vidéo d'introduction fantastique sur le PE : https://youtube.com/watch?v=dOxUroR57xs&feature=shares
```

> Un très bon guide d'ingénierie de prompts courts : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api

Reviewer: per style guide in [the Notion that Brasé set up (see Discord)](https://discord.com/channels/1046228027434086460/1080591486741118996/1080597776196448327), we're keeping terms like "`Prompt Engineering`" in English (same with "`PE`", like on line 9) - the rationale is in the Notion --> Traduction --> Règles de traduction --> Traduction des textes. I guess it's for recognizability? and i suppose we should at least be consistent within the learnprompting.org course itself. Otherwise "`ingénierie de prompts`" could also be translated alternatively as "`ingénierie des invites`" or "`ingénierie d'invites`", which I've seen online. 

```suggestion
Un très bon guide court sur le Prompt Engineering : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api
```

(similar feedback for the 5 other instances of "`ingénierie de prompts`")

maybe [if we prompt ChatGPT to generate translations (see Discord chat)](https://discord.com/channels/1046228027434086460/1080591486741118996/1089918144748998787), it'd be good to include this:

```txt
Do not translate "Prompt Engineering" or "PE" into French, but keep it as "Prompt Engineering" and "PE", respectively. 
```

Reviewer: > per style guide in [the Notion that Brasé set up (see Discord)](https://discord.com/channels/1046228027434086460/1080591486741118996/1080597776196448327), we're keeping terms like "`Prompt Engineering`" in English (same with "`PE`", like on line 9) - the rationale is in the Notion --> Traduction --> Règles de traduction --> Traduction des textes. I guess it's for recognizability? and i suppose we should at least be consistent within the learnprompting.org course itself. Otherwise "`ingénierie de prompts`" could also be translated alternatively as "`ingénierie des invites`" or "`ingénierie d'invites`", which I've seen online.
> 
> (similar feedback for the 5 other instances of "`ingénierie de prompts`")
> 
> maybe [if we prompt ChatGPT to generate translations (see Discord chat)](https://discord.com/channels/1046228027434086460/1080591486741118996/1089918144748998787), it'd be good to include this:
> 
> ```
> Do not translate "Prompt Engineering" or "PE" into French, but keep it as "Prompt Engineering" and "PE", respectively. 
> ```

@dr-nefardio3058 the other 5 instances of this in the file should also be fixed - i also added a suggestion for what to include in the prompt next time

> Une excellente introduction à l'ingénierie de prompts : https://humanloop.com/blog/prompt-engineering-101

Reviewer: ```suggestion
Une excellente introduction au Prompt Engineering : https://humanloop.com/blog/prompt-engineering-101
```

> Une belle collection de documents d'ingénierie de prompts : https://github.com/dair-ai/Prompt-Engineering-Guide

Reviewer: ```suggestion
Une belle collection de documents de Prompt Engineering : https://github.com/dair-ai/Prompt-Engineering-Guide
```

> De nombreux autres documents d'ingénierie de prompts : https://github.com/thunlp/PromptPapers

Reviewer: ```suggestion
De nombreux autres documents de Prompt Engineering : https://github.com/thunlp/PromptPapers
```

> Cours CMU sur l'ingénierie de prompts : https://youtu.be/5ef83Wljm-M

Reviewer: ```suggestion
Cours CMU sur le Prompt Engineering : https://youtu.be/5ef83Wljm-M
```

Comment fonctionne Copilot : https://thakkarparth007.github.io/copilot-explorer/posts/copilot-internals.html

Un guide de débutants pour l'écriture de prompts de Zapier : https://zapier.com/blog/gpt-3-prompt/

Une liste de ressources impressionnante sur l'ingénierie des prompts de A à Z : https://github.com/promptslab/Awesome-Prompt-Engineering

> 

Reviewer: If you re-run the translation, note that the English has extra lines here now https://learnprompting.org/docs/additional

> Une liste de ressources impressionnante sur l'ingénierie des prompts de A à Z : https://github.com/promptslab/Awesome-Prompt-Engineering

Reviewer: ```suggestion
Une liste de ressources impressionnante sur le Prompt Engineering de A à Z : https://github.com/promptslab/Awesome-Prompt-Engineering
```

> 

Reviewer: (the English has 4 extra links https://learnprompting.org/docs/additional )

```suggestion
500 modèles de prompts ChatGPT : https://www.notion.so/500-ChatGPT-Prompt-Templates-d9541e901b2b4e8f800e819bdc0256da

Prompt base: https://promptbase.com/

Vitrine de la communauté Midjourney (avec prompts) : https://www.midjourney.com/showcase/recent/

TOUS les styles et références pour Midjourney : https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference
```

### Exemples / Idées de prompts GPT-3

https://sharegpt.com 

https://www.learngpt.com 

https://chatgpt.getlaunchlist.com 

https://prompts.chat

## Faits

GPT-3 *n'est* PAS déterministe : https://twitter.com/BorisMPower/status/1608522707372740609

## Personnes à suivre

Je trouve du contenu significatif auprès de ces personnes.

> [@chillzaza_](https://mobile.twitter.com/chillzaza_) Ingénierie de prompts, outils, toolbot

Reviewer: ```suggestion
[@chillzaza_](https://mobile.twitter.com/chillzaza_) Prompt Engineering, outils, toolbot
```

> [@mathemagic1an](https://mobile.twitter.com/mathemagic1an) Diverses incitations, informations sur PE, GPT3

Reviewer: Various prompting --> not Diverses incitations, but rather maybe Prompts divers

```suggestion
[@mathemagic1an](https://mobile.twitter.com/mathemagic1an) Prompts divers, informations sur PE, GPT3
```

[@goodside](https://twitter.com/goodside/status/1588247865503010816) Injection de prompts, outillage PE

[@hwchase17](https://twitter.com/hwchase17) Dév principal de Langchain

[@omarsar0](https://twitter.com/omarsar0) Lead DAIR AI

> Veuillez également vérifier les comptes que je suis : https://twitter.com/learnprompting/following 

Reviewer: The original English was unfortunately "check" but actually means to say "check out":

```suggestion
Voir aussi les comptes que je suis : https://twitter.com/learnprompting/following 
```

"`Veuillez également ...`" sounds way too formal to my ears haha - and the original English didn't say "Please"

## Encore Plus

> Vérifiez les [problèmes ouverts](https://github.com/trigaten/Learn_Prompting/issues) et les [PRs](https://github.com/trigaten/Learn_Prompting/pulls) :)

Reviewer: Again, "check out" as in have a look, not the verifying kind of "check": 

```suggestion
Consultez les [problèmes ouverts](https://github.com/trigaten/Learn_Prompting/issues) et les [PRs](https://github.com/trigaten/Learn_Prompting/pulls) :)
```
